build/Dockerfile

```diff
@@ -2,12 +2,7 @@
 #                                     BUILD                                    #
 ################################################################################
 
-FROM gcr.io/cloud-marketplace/google/ubuntu1804:latest AS build
-
-ENV CLOUD_SDK_VERSION=340.0.0
-
-RUN apt-get update                                                          && \
-    apt-get -qq -y install git maven python
+FROM maven:3.8.4-eclipse-temurin-17-alpine AS build
```

Comment:
- maven uses the Apache License v2
- eclipse-temurin uses the Apache License v2 and the GNU General Public License, version 2 (GPL2) for OpenJDK
- Alpine uses the MIT License

These are all already used by other components in this repository, and hence pre-approved.